scripts/ingest_data.py

```diff
@@ -0,0 +1,17 @@
+import argparse
+from house_package_saachi.data_ingestion import fetch_housing_data, load_housing_data
```

Comment:
You can delete the `main.py` file because these 3 scripts are there to replace it.